src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5559,18 +5608,18 @@ def _build_onedeploy_arm_url(params):
     return params.cmd.cli_ctx.cloud.endpoints.resource_manager + base_url
 
 
-def _build_deploymentstatus_url(params, deployment_id):
+def _build_deploymentstatus_url(cmd, resource_group_name, webapp_name, slot, deployment_id):
```

Comment:
previously, all the values were part of this onedeploy `params` object, however since we want to reuse this method for non-onedeploy scenarios, only passing the required values to this method